crates/rolldown_std_utils/src/pretty_type_name.rs

```diff
@@ -1,17 +1,57 @@
 use std::borrow::Cow;
 
-use regex::Regex;
-use std::sync::LazyLock;
-
-static MODULE_MATCHER_RE: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"(?:\w+::)").unwrap());
```

Comment:
This is only used for debugging purpose. Normally this code won't be ran during bundling.

So maybe regex is more easier to maintain?

Reply:
For sure, I'll close it